src/app/submission/sections/duplicates/section-duplicates.component.html

```diff
@@ -0,0 +1,20 @@
+<!--
+Template for the detect duplicates submission section component
+@author Kim Shepherd
+-->
+<div class="text-sm-left" *ngVar="(this.getDuplicateData() | async) as data">
+  <ng-container *ngIf="data.potentialDuplicates.length == 0">
+    <p>{{ 'submission.sections.duplicates.none' }}</p>
```

Comment:
Here it should be `{{ 'submission.sections.duplicates.none' | translate }}`
instead of `{{ 'submission.sections.duplicates.none' }}`. At the moment the key is displayed, not the message.

Reply:
Fixed